package/kernel/mac80211/realtek.mk

```
@@ -201,6 +204,15 @@ define KernelPackage/rtw88-pci
   HIDDEN:=1
 endef
 
+define KernelPackage/rtw88-sdio
+  $(call KernelPackage/mac80211/Default)
+  TITLE:=Realtek RTW88 SDIO chips support
+  DEPENDS+= +kmod-rtw88
```

```suggestion
  DEPENDS+= +kmod-mmc +kmod-rtw88
```

The CI system found the following error:
```
Package kmod-rtw88-sdio is missing dependencies for the following libraries:
mmc_core.ko
make[3]: *** [Makefile:405: /__w/openwrt/openwrt/openwrt/bin/targets/malta/be/packages/kmod-rtw88-sdio_6.6.43.6.9.9-r1_mips_24kc.ipk] Error 1
time: package/kernel/mac80211/regular/compile#798.38#118.47#425.70
Error: Process completed with exit code 2.
```

---

fixed, thanks

---

strange, mmc_core and mmc_block kernel modules are not loaded (not exist in rootfs)

---

`grep -r sdio target/linux/malta` doesn't return anything. is CI system use special configuration?

---

For the rockchip target the MMC core is build into the kernel, then the kmod-mmc is empty. For the malta target the MMC core is not build into the kernel, then it is build as a module and packaged in kmod-mmc.

---

I see. then, is my last fix okay?